README.md

```diff
@@ -66,3 +66,25 @@ const mySchema = new Schema({
   marks
 });
 ```
+### `Yalc`: An alternative to yarn link - Testing with flexible-content locally
+A very simple local repository for your locally developed packages that you want to share across your local environment.
```

Comment:
This description is quite verbose: let's be as concise as we can. I'd be tempted to rewrite everything between L70 and L76 as:

_> We've found [yalc](https://github.com/wclr/yalc) useful in testing local changes to `prosemirror-elements` in applications that use it._